src/Pages/Overview/Overview.js

```diff
@@ -2,15 +2,24 @@ import React, { Component } from 'react'
 import './Overview.css'
 import OverviewProfile from './OverviewProfile.js'
 import { getAllUsers, deleteUserByEmail } from '../../APIFunctions/User'
+import {
+  ButtonDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem
+} from 'reactstrap'
+import { membershipState } from '../../Enums'
 
 export default class OverviewBoard extends Component {
   constructor (props) {
     super(props)
     this.state = {
       // All users array, update by callDatabase
       users: [],
```

Comment:
your fixing of the update bug good, can you open a seperate PR for the search types?

Reply:
It a pain trying to separate them now. So I woudnt like that

Reply:
Its not much, so can you look at them